pyaedt/generic/general_methods.py

```diff
@@ -1007,9 +1008,166 @@ def _create_json_file(json_dict, full_json_path):
     return True
 
 
+# @pyaedt_function_handler()
+# def com_active_sessions(version=None, student_version=False, non_graphical=False):
+#     """Get information for the active COM AEDT sessions.
+#
+#     Parameters
+#     ----------
+#     version : str, optional
+#         Version to check. The default is ``None``, in which case all versions are checked.
+#         When specifying a version, you can use a three-digit format like ``"222"`` or a
+#         five-digit format like ``"2022.2"``.
+#     student_version : bool, optional
+#         Whether to check for student version sessions. The default is ``False``.
```

Comment:
Someday I'll try to standardize on either "Student Version" or "student version," but I don't know which (though I prefer the lower case).